core/src/test/java/org/hisp/dhis/android/core/arch/api/filters/internal/FilterConverterShould.java

```diff
@@ -141,58 +141,6 @@ public void returns_correct_path_when_create_a_retrofit_request_using_filters_an
                 "/api?field=id,code,name,displayName&filter=id:in:[uid1,uid2]&filter=lastUpdated:gt:updatedDate");
     }
 
-    @Test
-    public void returns_correct_path_when_create_a_retrofit_request_ignoring_null_filter() throws IOException, InterruptedException {
-        ArrayList<String> values = new ArrayList<>(2);
-        values.add("uid1");
-        values.add("uid2");
-
-        TestService service = testService(server);
-
-        service.test(
-                InFilter.create(Field.create("id"), values),
-                SingleValueFilter.gt(Field.create("lastUpdated"), null)
```

Comment:
I think this test still makes sense. This second property could be null to prove that a null Filter is ignored in the generated url.